web/modules/custom/owner/owner.install

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * @file
+ * Install, uninstall, schema and requirements hook for the owner module.
+ */
+
+/**
+ * Implements hook_update_N().
```

Comment:
We should drop this like in the other hooks, so the update list doesn't contain this unneeded information.